log.txt

```diff
@@ -0,0 +1,2732 @@
+  DESCEND  objtool
```

Comment:
You probably don't want to keep this log file in Git.
It can be removed from the index with `git rm --cached log.txt`